Sources/MongoKitten/AggregateStage.swift

```diff
@@ -206,4 +206,58 @@ public struct AggregateBuilderStage {
+    ///   - maxDistance: The maximum distance from the center point that the documents can be. MongoDB limits the results to those documents that fall within the specified distance from the center point. Specify the distance in meters if the specified point is GeoJSON and in radians if the specified point is legacy coordinate pairs.
+    ///   - query: Optional. Limits the results to the documents that match the query. The query syntax is the usual MongoDB read operation query syntax. You cannot specify a $near predicate in the query field of the $geoNear stage.
+    ///   - distanceMultiplier: Optional. The factor to multiply all distances returned by the query. For example, use the distanceMultiplier to convert radians, as returned by a spherical query, to kilometers by multiplying by the radius of the Earth.
+    ///   - includeLocs: Optional. This specifies the output field that identifies the location used to calculate the distance. This option is useful when a location field contains multiple locations. To specify a field within an embedded document, use dot notation.
+    ///   - uniqueDocs: Optional. If this value is true, the query returns a matching document once, even if more than one of the document’s location fields match the query.
+    ///   - minDistance: Optional. The minimum distance from the center point that the documents can be. MongoDB limits the results to those documents that fall outside the specified distance from the center point.
+    ///   - key: Specify the geospatial indexed field to use when calculating the distance. If your collection has multiple 2d and/or multiple 2dsphere indexes, you must use the key option to specify the indexed field path to use. Specify Which Geospatial Index to Use provides a full example. If there is more than one 2d index or more than one 2dsphere index and you do not specify a key, MongoDB will return an error. If you do not specify the key, and you have at most only one 2d index and/or only one 2dsphere index, MongoDB looks first for a 2d index to use. If a 2d index does not exists, then MongoDB looks for a 2dsphere index to use.
+    public static func geoNear(
+        useLegacy: Bool = false,
+        coordinates: [Double],
+        distanceField: String,
+        spherical: Bool = false,
+        maxDistance: Int? = nil,
+        query: Document? = nil,
+        distanceMultiplier: Int? = nil,
+        includeLocs: String? = nil,
+        uniqueDocs: Bool? = nil,
+        minDistance: Int? = nil,
+        key: String? = nil
+    ) -> AggregateBuilderStage {
+        var geoNear: Document = ["distanceField": distanceField,
+                                 "spherical": spherical]
+        
+        if useLegacy {
+            geoNear["near"] = coordinates.makePrimitive()
+        } else {
+            geoNear["near"] = ["type": "Point", "coordinates": coordinates] as Document
```

Comment:
I assume that geoNear only supports a `point`, right?
Looking at the API a second time makes me wonder whether `useLegacy` should be a boolean option. 
Also, coordinates is _either_ 2 or 3 coordinates if I remember correctly. In which case the `[Double]` is fine. But I'm wondering if that API could be changed to be less prone to error. A good example of an error that I keep making is the order of `latitude` and `longitude`.

I also wonder whether `uniqueDocs` should be written out fully as `uniqueDocuments`, irregardless of MongoDB's documentation.

These are not _decisions_, but something I'd wish to discuss before merging for sure.

Reply:
Also updated these.